common/models/User.php

```diff
@@ -89,6 +89,7 @@ public function rules()
             [['username', 'email'], 'unique'],
             ['status', 'default', 'value' => self::STATUS_ACTIVE],
             ['status', 'in', 'range' => [self::STATUS_ACTIVE, self::STATUS_DELETED]],
+            [['username','email','password_hash'],'filter','filter'=>'\yii\helpers\Html::encode']
```

Comment:
EmailValidator for email should be enough to prevent xss.
And remove password_hash from this rule :-)